geometry_msgs/TransformStamped pose
---
bool calibrated